sktm/patchwork.py

```diff
@@ -291,13 +291,13 @@ def __init__(self, baseurl, project_name, skip, is_rh_fork=False):
                           default.
         """
         self.baseurl = baseurl
-        self.project_id = self.get_project_id(project_name)
+        self.project_id = self.__get_project_id(project_name)
```

Comment:
This method is defined in child classes so I believe this change would break things.